tracer/test/snapshots/WcfTests.__binding=WSHttpBinding_enableNewWcfInstrumentation=True.verified.txt

```diff
@@ -3,7 +3,7 @@
     TraceId: Id_1,
     SpanId: Id_2,
     Name: wcf.request,
-    Resource: WcfSample/ICalculator/ServerSyncAdd,
+    Resource: /WcfSample/?/CalculatorService,
```

Comment:
Just so I understand, this one has the URL rather than a useful name because it's the one with `Action=""` right? I wonder if we should strip the leading `/` from the URL, just so it looks _similar_ to the ones with an action? Or maybe it's better that it's different, so it can't be confused for the others? 🤔 Open to discussion!

Reply:
Correct. For the `Action=""` it falls back to using the `LocalPath` for the resource name (also I couldn't figure out how to get `Action` to be `null` without the server throwing).

Making them look similar would make sense to me - since we do lose the "useful" name maybe it's better to not though? If anyone asks why they aren't getting the Action in the resource name and they have that leading `/` we know that the Action was actually empty or null and they were falling back to the URL form? :)

I didn't think anything of that leading `/` so I'd be fine either stripping it out or leaving it in

Reply:
I would prefer keeping the leading `/` to help differentiate between an action and a path. For `aspnet.request` spans we do something similar for the resource names, where we give it a pattern `<HTTP_METHOD> <PATH>` where path has a leading `/`, so this would be consistent

Reply:
nit: One thing you could do to minimize the diff in the snapshots is add the new test case to the end so the bulk of the changes are in the last span in the file, but definitely not a requirement